src/transformers.ts

```diff
@@ -12,8 +13,15 @@ export interface TransformerConfig {
   drop?: KeyTarget
   sample?: TransformerConfigSample
   map?: Record<string, TransformerConfigMap>
+  encrypt?: EncryptConfig
 }
 
+export interface EncryptConfig{
```

Comment:
These keys should be lowercase per convention

Reply:
but all other interfaces are also similar

Reply:
Not sure I follow. Also, there should be a space here `EncryptConfig {`.

Reply:
```
export interface TransformerConfig {
  allow?: KeyTarget
  drop?: KeyTarget
  sample?: TransformerConfigSample
  map?: Record<string, TransformerConfigMap>
  encrypt?: EncryptConfig
}

export interface EncryptConfig {
  Key: string
  Properties: string[]
  Label: string
  Seed: string
}
export interface TransformerConfigSample {
  percent: number
  path: string
}

export interface TransformerConfigMap {
  set?: any
  copy?: string
  move?: string
  to_string?: boolean
}
```
all other configs are camelCase.
i can fix that

Reply:
oh yes, by lower case I just meant "follow the existing convention" =)

Reply:
ah got it the the values in the interface :)